documentation/staging/content/managing-domains/accessing-the-domain/domain-events.md

```diff
@@ -29,8 +29,11 @@ The domain events have been enhanced in 4.0. Here is a summary of the changes in
     * Have a better failure categorization (see [Operator-generated event types](#operator-generated-event-types) for more details).
     * Include the categorization information in the event message.
     * Provide more information in the event message to indicate what has gone wrong, what you need to do to resolve the problem, and if the operator will [retry]({{< relref "/managing-domains/domain-lifecycle/retry.md" >}}) the failed operation.
-* Added three event types, `Unavailable`, `Incomplete`, and `FailureResolved`, to record the transition of [domain condition]({{< relref "/managing-domains/accessing-the-domain/status-conditions.md" >}})
-  `Available/Completed/Failed` from `True` to `False`, and vice versa.
+* Added four event types, `Available`, `Unavailable`, `Incomplete`, and `FailureResolved`, to record
+  the transition of their corresponding [domain resource status conditions]({{< relref "/managing-domains/accessing-the-domain/status-conditions#types-of-domain-conditions" >}}).
+* Added seven event types, `ClusterAvailable`, `ClusterChanged`, `ClusterCompleted`, `ClusterCreated`,
+  `ClusterDeleted`, `ClusterIncomplete`, and `ClusterUnavailable`, to record the transition of their
+  corresponding [cluster resource status conditions]({{< relref "/managing-domains/accessing-the-domain/status-conditions#types-of-cluster-conditions" >}}).
```

Comment:
Minor: How are ClusterCreated and ClusterDeleted reflected by cluster conditions? I suppose the first is reflected by the existence of the Available and Completed conditions in a valid cluster resource, and the non-existence of the Available and Completed conditions in an invalid cluster resource. The former has been tested, but I don't know about the latter. Overall though, this seems pretty subtle to me.

Reply:
I think in an invalid cluster resource, both Available and Completed conditions would be false.

Reply:
That's better. Good enough IMO.